### DISCUSSION TITLE
addHeading

### DISCUSSION THREAD

**Discussion by rotkiv1205** [jesper-raemaekers/python-polarion#117] [Q&A] [answered]
addHeading adds a heading with textsize being "Title". I want to add a text with "Heading 2" that gets added to my document outline. Should I still use the addHeading or how do i go on about this? :)

Edit:
Found [documentation ](https://python-polarion.readthedocs.io/en/latest/document.html#document-headings)_**after**_ answer.

**Comment by jesper-raemaekers** [comment, marked as answer]
Yeah you still use the same function but you add a parent to it. A child of title will be heading 1 and a child of that will be heading 2 and so on..

```
h1 = doc.addHeading('Header 1')
h2 = doc.addHeading('Header 2', h1)
h3 = doc.addHeading('Header 3', h2)
```
